tests/headers/issue-1554.h

```diff
@@ -0,0 +1,6 @@
+// bindgen-flags: --default-enum-style rust_non_exhaustive
+
+enum Planet {
```

Comment:
This test won't pass on automation, you probably need: `--raw-line '#![cfg(feature = "nightly")]' --raw-line '#![feature(non_exhaustive)]'` on the flags above.